src/EnergyPlus/DesiccantDehumidifiers.cc

```diff
@@ -232,48 +212,13 @@ namespace DesiccantDehumidifiers {
         //                        Work supported by ASHRAE research project 1254-RP
         //                      June 2007 R. Raustad, FSEC
         //                        Added new dehumidifier type -- DESICCANT DEHUMIDIFIER
-        //       RE-ENGINEERED  na
 
         // PURPOSE OF THIS SUBROUTINE:
         // Obtains input data for humidifiers and stores it in dehumidifier data structures.
 
         // METHODOLOGY EMPLOYED:
         // Uses InputProcessor "Get" routines to obtain data.
 
-        // Using/Aliasing
-        using BranchNodeConnections::SetUpCompSets;
-        using BranchNodeConnections::TestCompSet;
-        using NodeInputManager::GetOnlySingleNode;
-        auto &GetDXCoilOutletNode(DXCoils::GetCoilOutletNode);
```

Comment:
This is an obnoxious use of `auto` that I am trying to get rid of (among the other ones).